src/mumble/main.cpp

```diff
@@ -221,227 +223,189 @@ int main(int argc, char **argv) {
+																  "Mumble's settings.\n");
+
+	app.add_flag("-m, --multiple", bAllowMultiple, "Allow multiple instances of the client to be started.\n");
+	app.add_flag("-n, --noidentity", suppressIdentity, "Suppress loading of identity files (i.e., certificates.)\n");
+	app.add_flag("--license", print_license, "Show the Mumble license.\n");
+	app.add_flag("--authors", print_authors, "Show the Mumble authors.\n");
+	app.add_flag("--third-party-licenses", print_third_party_licenses,
+				 "Show licenses for third-party software used by Mumble.\n");
+	app.add_flag("--dump-input-streams", dump_input_streams,
+				 "Dump PCM streams at various parts of the input chain\n"
+				 "(useful for debugging purposes)\n"
+				 "- raw microphone input\n"
+				 "- speaker readback for echo cancelling\n"
+				 "- processed microphone input\n");
+	app.add_flag("--print-echocancel-queue", print_echocancel_queue,
+				 "Print on stdout the echo cancellation queue state\n"
+				 "(useful for debugging purposes)\n");
+	app.add_flag("--print-translation-dirs", printTranslationDirs,
+				 "Print out the paths in which Mumble will search for\n"
+				 "translation files that overwrite the bundled ones.\n"
+				 "(Useful for translators testing their translations)\n");
+	app.add_flag("--version", print_version, "Print version information and exit");
+
+	CLI::App *rpc_subcommand = app.add_subcommand("rpc", "It is possible to remote control a running instance\n"
+														 "of Mumble by using the 'mumble rpc' command.\n"
+														 "\n"
+														 "Valid actions are:\n"
+														 "  mute\n"
+														 "                Mute self\n"
+														 "  unmute\n"
+														 "                Unmute self\n"
+														 "  togglemute\n"
+														 "                Toggle self-mute status\n"
+														 "  deaf\n"
+														 "                Deafen self\n"
+														 "  undeaf\n"
+														 "                Undeafen self\n"
+														 "  toggledeaf\n"
+														 "                Toggle self-deafen status\n"
+														 "  starttalking\n"
+														 "                Start talking\n"
+														 "  stoptalking\n"
+														 "                Stop talking\n");
+
+	CLI::Option *option_rpc_command = rpc_subcommand->add_option("rpcCommand", rpcCommand);
+
+	CLI11_PARSE(app, argc, argv);
+
+  if (*rpc_subcommand) {
+    bRpcMode = true;
+    if (!(*option_rpc_command)) {
+      QString rpcError = MainWindow::tr("Error: No RPC command specified");
 #if defined(Q_OS_WIN)
-				QMessageBox::information(nullptr, MainWindow::tr("Invocation"), helpOutput);
+      QMessageBox::information(nullptr, MainWindow::tr("RPC"), rpcError);
 #else
-				printf("%s", qPrintable(helpOutput));
+      printf("%s\n", qPrintable(rpcError));
 #endif
+      return 1;
+    }
+  }
+
+	if (*option_jackname) {
+		if (custom_jack_client_name.empty()) {
+			qCritical("Missing argument for --jackname!");
+			return 1;
```

Comment:
To my understanding, this shouldn't be needed. At least, if we manage to set up CLI11 correctly. That's because then CLI11 should know that this option requires a value and therefore error itself during the CLI argument parsing stage, meaning that if the option was used, it'll always have a value set.
In that case, it would also seem like we don't have to use the option pointers as we can simply check whether the corresponding variable has a non-empty value.